programs/obfuscator/Obfuscator.cpp

```diff
@@ -1238,6 +1238,7 @@ try
     po::store(parsed, options);
 
     if (options.count("help")
+        || (options.count("host") && options["host"].as<std::string>() == "elp")
```

Comment:
This seems to detect a typo. Which one exactly? To be honest, handling a particular typo begs the question why we should not have logic for other typos too. Guess I favor not handling typos at all.

EDIT: Got it. Comment `/// If user writes -help instead of --help.` is missing. (also other files).

Well, I think 1. the logic to detect `-help` looks weird (e.g. will it break if - for whatever reason - parameter `--host` is removed at some point) 2. it makes the CLI inconsistent in the sense that a long option with a single dash is introduced. It has some charm but since consistency is king, I would actually prefer to omit it.